webapp/components/Editor/MenuLegend.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <dropdown class="content-menu" :placement="placement" offset="5">
+    <template #default="{ toggleMenu }">
+      <slot name="button" :toggleMenu="toggleMenu">
+        <base-button
+          data-test="content-menu-button"
+          icon="question-circle"
+          size="small"
+          circle
+          ghost
+          @click="toggleMenu()"
+        />
+      </slot>
+    </template>
+    <template #popover="" class="legend">
+      <div class="legend-container">
+        <div class="legend-header">Keyboard shortcuts and markdown code</div>
+        <div class="legend-table" v-for="item in legenditems" :key="item.name">
+          <base-button size="small" circle ghost :icon="item.iconname" class="legend-icon">
+            <span v-if="item.label">{{ item.label }}</span>
+          </base-button>
+          <span>{{ item.name }}</span>
+          <span class="tool-shortcut">{{ item.shortcut }}</span>
+        </div>
+      </div>
+    </template>
+  </dropdown>
+</template>
+
+<script>
+import Dropdown from '~/components/Dropdown'
+
+export default {
+  components: {
+    Dropdown,
+  },
+  props: {
+    placement: { type: String, default: 'bottom-start' },
+  },
+  data() {
+    return {
+      hover: false,
+      legenditems: [
+        { iconname: 'bold', name: 'bold', shortcut: 'Ctrl+b' },
+        { iconname: 'italic', name: 'italic', shortcut: 'Ctrl+i' },
+        { iconname: 'underline', name: 'underline', shortcut: 'Ctrl+u' },
+        { iconname: 'link', name: 'link', shortcut: '' },
+        { iconname: 'paragraph', name: 'paragraph', shortcut: '' },
+        { label: 'H3', name: 'heading 3', shortcut: '### + space' },
+        { label: 'H4', name: 'heading 4', shortcut: '#### + space' },
+        { iconname: 'list-ul', name: 'unordered list', shortcut: '* + space' },
+        { iconname: 'list-ol', name: 'ordered list', shortcut: '1. + space' },
+        { iconname: 'quote-right', name: 'quote', shortcut: '> + space' },
```

Comment:
Zitat

Reply:
> I gave you the German translations for the legend. What happens, if you run `yarn locales --fix` in the webapp folder?

Thanks for the German translations! `yarn locales --fix` runs two scripts in scripts/translations, they both result in "jq: command not found". I tried removing jq from the main package.json file and adding the dependency just to the webapp but it gives the same message.

Reply:
Could this be related? https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/master/.github/workflows/test.yml#L25
I am not sure about all those translation tools & checks, but we have two scripts executed in our test workflow.
@Mogge